pimoroni_display_hat_mini.h

```diff
@@ -0,0 +1,22 @@
+#pragma once
+
+// Data specific to the Pimoroni Display HAT mini
+// 320x240 2.0" ST7789V2 HAT
+// https://shop.pimoroni.com/products/display-hat-mini
+
+#ifdef PIMORONI_DISPLAY_HAT_MINI

```

Comment:
Might there ever be mini display hats from Pimoroni based on other controller technology? If there's a possibility, then maybe this might better be `#ifdef PIMORONI_ST7789_DISPLAY_HAT_MINI`?

If not, then the name lgtm.

Reply:
That's a possibility. Good point!